data/clippings.txt

```diff
@@ -373,3 +373,113 @@ You get in life what you have the courage to ask for.
 
 Just imagine how would you look like in 6 months if you don't give up ...
 ==========
+Psicología del éxito (Mario Luna)
```

Comment:
Please replace the first line of each quote with the formatting generalization used in order to be merged:
- _Book's name - Author's name_

Instead of yours:
- _Book's name (Author's name)_

Reply:
@Rodrigo-Flores PD: You could use the same Python, to replace each required symbol of a given file, for the sake of time.